Gopkg.toml

```diff
@@ -7,7 +7,7 @@
   non-go = true
 
   [[prune.project]]
```

Comment:
client-go is pure go project now. We can remove it.

Reply:
:+1: done in 8f25ae5